wirecard/oxid-ee
# WDPD-202 Improve Payolution configuration

### This PR

* Improves configuration handling for Wirecard Guaranteed Invoice (Payolution) payment method in admin panel.
* If "Require consent" is set to "Yes", "Payolution URL" is mandatory (otherwise it can be empty).
* If new currency is added, an information that "currency configuration fields should be filled" is shown.

### How to Test

* Go to ``Shop settings -> Payment Methods`` and select "Wirecard Guaranteed Invoice (Payolution)".
* Set "Require consent" to "Yes", and keep "Payolution URL" empty. Click "Save". An error with configuration should be shown. 
* Select a new currency and click save. An information that "currency configuration fields should be filled" should be shown.

### Jira Links
https://jira.parkside.at/browse/WDPD-202
